aggregator/templates/includes/badge-card.html

```diff
@@ -0,0 +1,5 @@
+{% load staticfiles %}
+<div class="card {{ class }}">
+  <img src="{% static path %}">
+  <code>https://developers.democracyclub.org.uk{% static path %}</code>
```

Comment:
Any reason not to use `request.META.HTTP_HOST` here? I'm pretty sure we've already done the requisite CloudFront header fiddling to get internal links to work properly.